Readme.md

```diff
@@ -13,18 +13,23 @@ Using the task is simple. The only mandatory attribute is url and the default HT
 
 	<?xml version="1.0"?>
 	<project name="Http">
-	  <http url="http://www.howtocreate.co.uk/operaStuff/userjs/samplexml.xml"
+	  <http url="http://httpbin.org/get"
 	        method="GET"
-	        contenttype="text/xml"
+	        contenttype="application/json"
 	        connectiontimeout="30"
 	        responseproperty="response"
 	        statuscodeproperty="status"
-	        failonerror="true" />
+	        failonerror="true" 
+	        username="jdoe"
+	        password="53cr34"
+	        content="{ &quot;greeting}&quot; : &quot;Hello world!&quot; }" />
```

Comment:
Hmm ... true. I'd also need to change the URL to http://httpbin.org/post to keep internal consistency.